tests/attr/layer/test_layer_lrp.py

```diff
@@ -0,0 +1,173 @@
+        relevance_default = lrp_default.attribute(inputs, attribute_to_layer_input=True)
+        relevance_inplace = lrp_inplace.attribute(inputs, attribute_to_layer_input=True)
+        assertTensorAlmostEqual(self, relevance_default, relevance_inplace)
+
+    def test_lrp_simple_tanh(self):
+        class Model(nn.Module):
+            def __init__(self):
+                super(Model, self).__init__()
+                self.linear = nn.Linear(3, 3, bias=False)
+                self.linear.weight.data.fill_(0.1)
+                self.tanh = torch.nn.Tanh()
+                self.linear2 = nn.Linear(3, 1, bias=False)
+                self.linear2.weight.data.fill_(0.1)
+
+            def forward(self, x):
+                return self.linear2(self.tanh(self.linear(x)))
+
+        model = Model()
+        _, inputs = _get_simple_model()
+        lrp = LayerLRP(model, model.linear)
+        relevance = lrp.attribute(inputs)
+        assertTensorAlmostEqual(
+            self, relevance, torch.Tensor([[1 / 3, 1 / 3, 1 / 3]])
+        )  # Result if tanh is skipped for propagation
+
+    def test_lrp_simple_attributions_GammaRule(self):
+        model, inputs = _get_simple_model()
+        with torch.no_grad():
+            model.linear.weight.data[0][0] = -2
+        model.eval()
+        model.linear.rule = GammaRule(gamma=1)
+        model.linear2.rule = GammaRule()
+        lrp = LayerLRP(model, model.linear)
+        relevance = lrp.attribute(inputs)
+        assertTensorAlmostEqual(
+            self, relevance.data, torch.tensor([0.2500, 0.3750, 0.3750])
+        )
+
+    def test_lrp_simple_attributions_AlphaBeta(self):
+        model, inputs = _get_simple_model()
+        with torch.no_grad():
+            model.linear.weight.data[0][0] = -2
+        model.eval()
+        model.linear.rule = Alpha1_Beta0_Rule()
+        model.linear2.rule = Alpha1_Beta0_Rule()
+        lrp = LayerLRP(model, model.linear)
+        relevance = lrp.attribute(inputs)
+        assertTensorAlmostEqual(self, relevance, torch.tensor([0.2500, 0.3750, 0.3750]))
+
+    def test_lrp_simple_attributions_all_layers(self):
+        model, inputs = _get_simple_model(inplace=False)
+        model.eval()
+        model.linear.rule = EpsilonRule()
+        model.linear2.rule = EpsilonRule()
+        lrp = LayerLRP(model, None)
+        relevance = lrp.attribute(inputs, attribute_to_layer_input=True)
+        self.assertEqual(len(relevance), 2)
+
+    def test_lrp_simple_attributions_all_layers_delta(self):
+        model, inputs = _get_simple_model(inplace=False)
+        model.eval()
+        model.linear.rule = EpsilonRule()
+        model.linear2.rule = EpsilonRule()
+        lrp = LayerLRP(model, None)
+        inputs = torch.stack((inputs, 2 * inputs))
+        relevance, delta = lrp.attribute(
+            inputs, attribute_to_layer_input=True, return_convergence_delta=True
+        )
+        self.assertEqual(len(relevance), len(delta))
```

Comment:
nit: Can we, please, also assert the actual values instead of only the shapes ?

Reply:
The relevance values are checked now.